Projects/Features/ArtistFeature/Sources/ViewModels/ArtistMusicContentViewModel.swift

```diff
@@ -31,30 +31,26 @@ public final class ArtistMusicContentViewModel: ViewModelType {
     }
 
     public struct Input {
-        var pageID: BehaviorRelay<Int>
+        var pageID: BehaviorRelay<Int> = BehaviorRelay(value: 1)
         var songTapped: PublishSubject<Int> = PublishSubject()
         var allSongSelected: PublishSubject<Bool> = PublishSubject()
     }
 
     public struct Output {
-        var canLoadMore: BehaviorRelay<Bool>
-        var dataSource: BehaviorRelay<[ArtistSongListEntity]>
-        let indexOfSelectedSongs: BehaviorRelay<[Int]>
-        let songEntityOfSelectedSongs: BehaviorRelay<[SongEntity]>
+        var canLoadMore: BehaviorRelay<Bool> = BehaviorRelay(value: true)
+        var dataSource: BehaviorRelay<[ArtistSongListEntity]> = BehaviorRelay(value: [])
+        var indexOfSelectedSongs: BehaviorRelay<[Int]> = BehaviorRelay(value: [])
+        var songEntityOfSelectedSongs: BehaviorRelay<[SongEntity]> = BehaviorRelay(value: [])
```

Comment:
```suggestion
        let canLoadMore: BehaviorRelay<Bool> = BehaviorRelay(value: true)
        let dataSource: BehaviorRelay<[ArtistSongListEntity]> = BehaviorRelay(value: [])
        let indexOfSelectedSongs: BehaviorRelay<[Int]> = BehaviorRelay(value: [])
        let songEntityOfSelectedSongs: BehaviorRelay<[SongEntity]> = BehaviorRelay(value: [])
```
직접 바뀌는게 아니니 let으로 해도 괜찮을거같아요!